lib/LaTeXML/Core/Gullet.pm

```diff
@@ -101,6 +101,15 @@ sub flush {
   $$self{mouthstack} = [];
   return; }
 
+sub setup_scan {
+  my ($self) = @_;
+  $$self{pushback_has_smuggled_the} = 0;
+  # setup new scan by removing any smuggle CCs
+  for my $token (@{ $$self{pushback} }) {
+    if ($$token[1] == CC_SMUGGLE_THE) {
+      $token = $$token[2]; } }
```

Comment:
btw this for loop ought to be the fastest way to sneak out the catcode without needless overhead. Running a `map` that replaces the pushback, which is what I tried in my first exploratory commit, is noticeably slower.

Reply:
That actually *replaces* the tokens? darm perl!

Reply:
yeah, I had to double-check the docs to make sure. Rule of thumb is that using an iterator is both fastest+safest (in any lang), and apparently within one of these iterator-style loops the variable is an _alias_ for the array element.